debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -2717,21 +2775,24 @@ public void shouldNotAckLsnOnSource() throws Exception {
         Assert.assertEquals(slotAtTheBeginning.slotLastFlushedLsn(), slotAfterSnapshot.slotLastFlushedLsn());
 
         TestHelper.execute("INSERT INTO s2.a (aa,bb) VALUES (1, 'test');");
-        TestHelper.execute("UPDATE s2.a SET aa=2, bb='hello' WHERE pk=2;");
+        // YB note: since update records are not yet supported, commenting this and reducing the
```

Comment:
Could you mention the list of tests related to Updates that would need to be enabled once the Update support is made available in the ticket for Update support

Reply:
Done.